src/engine/runtime.js

```diff
@@ -1522,24 +1522,37 @@ class Runtime extends EventEmitter {
      * @param {!MonitorRecord} monitor Monitor to add.
```

Comment:
This comment should be updated to replace 'overwrites it.' with something like: 'updates those properties that are defined in the given monitor record.'